.github/workflows/ci.yml

```diff
@@ -308,3 +308,40 @@ jobs:
           BITCOINFUZZ: "${{ github.workspace}}\\src\\fuzz.exe"
         shell: cmd
         run: py -3 test\fuzz\test_runner.py --par %NUMBER_OF_PROCESSORS% --loglevel DEBUG %RUNNER_TEMP%\qa-assets\fuzz_seed_corpus
+
+  asan-lsan-ubsan-integer-no-depends-usdt:
+    name: 'ASan + LSan + UBSan + integer, no depends, USDT'
+    runs-on: ubuntu-24.04
+    # No need to run on the read-only mirror, unless it is a PR.
+    if: github.repository != 'bitcoin-core/gui' || github.event_name == 'pull_request'
+    timeout-minutes: 120
+    env:
+      FILE_ENV: "./ci/test/00_setup_env_native_asan.sh"
+      CIRRUS_CI: true
+      TEST_RUNNER_EXTRA: --exclude "rpc_bind.py --ipv6,feature_proxy.py"
+      MAKEJOBS: '-j1'
```

Comment:
Is a single job chosen for a reson here? The free hosted GHA runners have 4 (v)CPUs available AFAIK...

Reply:
I have been hitting issues on github actions runners cancelling the job after 4 minutes ,which is during compile, so this was an attempt to reduce the CPU load. It hasn't helped. See https://github.com/actions/runner-images/issues/9848#issuecomment-2137525256